File: src/HichemBundle/Entity/Ligneproduit.php
<?php

namespace HichemBundle\Entity;

use Doctrine\ORM\Mapping as ORM;

/**
 * Ligneproduit
 *
 * @ORM\Table(name="ligneproduit", indexes={@ORM\Index(name="idpanier", columns={"idpanier"}), @ORM\Index(name="idproduit", columns={"idproduit"})})
 * @ORM\Entity(repositoryClass="HichemBundle\Repository\LigneproduitRepository")
 */
class Ligneproduit
{
    /**
     * @var integer
     *
     * @ORM\Column(name="idligne", type="integer", nullable=false)
     * @ORM\Id
     * @ORM\GeneratedValue(strategy="IDENTITY")
     */
    private $idligne;

    /**
     * @var integer
     *
     * @ORM\Column(name="quantite", type="integer", nullable=false)
     */
    private $quantite;

 /*   /**
     * @var \Panier
     *
     * @ORM\ManyToOne(targetEntity="Panier")
     * @ORM\JoinColumns({
     *   @ORM\JoinColumn(name="idpanier", referencedColumnName="idpanier")
     * })
     */
    /**
     * @var int
     *
     * @ORM\Column(name="idpanier", type="integer", nullable=false)
     */


    private $idpanier;

    /**
     * @return int
     */
    public function getQuantite()
    {
        return $this->quantite;
    }

    /**
     * @param int $quantite
     */
    public function setQuantite($quantite)
    {
        $this->quantite = $quantite;
    }

   /* /**
     * @return \Panier
     */

    /**
     * @return int
     */
    public function getIdpanier()
    {
        return $this->idpanier;
    }

    /* /**
     * @param \Panier $idpanier
     */

    /**
     * @return int
     */

    public function setIdpanier($idpanier)
    {
        $this->idpanier = $idpanier;
    }

    /**
     * @return \Produit
     */
    public function getIdproduit()
    {
        return $this->idproduit;
    }

    /**
     * @param \Produit $idproduit
     */
    public function setIdproduit($idproduit)
    {
        $this->idproduit = $idproduit;
    }

    /**
     * @var \Produit
     *
     * @ORM\ManyToOne(targetEntity="Produit")
     * @ORM\JoinColumns({
     *   @ORM\JoinColumn(name="idproduit", referencedColumnName="idproduit")
     * })
     */
    private $idproduit;

    /**
     * @return int
     */
    public function getIdligne()
    {
        return $this->idligne;
    }

    /**
     * @param int $idligne
     */
    public function setIdligne($idligne)
    {
        $this->idligne = $idligne;
    }


}

File: Java/pepinierecomplet/src/entite/ligneproduit.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package entite;

/**
 *
 * @author Hichem
 */
public class ligneproduit {
    private int idligne;
    private int idpanier ;
    private int idproduit;
    private int quantite ;

    public ligneproduit() {
    }

    public ligneproduit(int idligne, int idpanier, int idproduit, int quantite) {
        this.idligne = idligne;
        this.idpanier = idpanier;
        this.idproduit = idproduit;
        this.quantite = quantite;
    }
     public ligneproduit(int idpanier, int idproduit, int quantite) {
      //  this.idligne = idligne;
        this.idpanier = idpanier;
        this.idproduit = idproduit;
        this.quantite = quantite;
    }

    public int getIdligne() {
        return idligne;
    }

    public void setIdligne(int idligne) {
        this.idligne = idligne;
    }

    public int getIdpanier() {
        return idpanier;
    }

    public void setIdpanier(int idpanier) {
        this.idpanier = idpanier;
    }

    public int getIdproduit() {
        return idproduit;
    }

    public void setIdproduit(int idproduit) {
        this.idproduit = idproduit;
    }

    public int getQuantite() {
        return quantite;
    }

    public void setQuantite(int quantite) {
        this.quantite = quantite;
    }

    @Override
    public String toString() {
        return "ligneproduit{" + "idligne=" + idligne + ", idpanier=" + idpanier + ", idproduit=" + idproduit + ", quantite=" + quantite + '}';
    }
    
}
